src/site/content/en/patterns/advanced-apps/badges/index.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: pattern
+title: How to create an app badge
+date: 2022-05-18
+description: |
+  Learn how to create an app badge.
+static:
+  - assets/manifest.json
+  - assets/sw.js
```

Comment:
I suggest you just move sw.js up a level to badges/sw.js and update the path here.